web/client/components/map/leaflet/Map.jsx

```diff
@@ -268,7 +268,8 @@ let LeafletMap = React.createClass({
         this.props.onMouseMove({
             x: pos.lng,
             y: pos.lat,
-            crs: "EPSG:4326"
+            crs: "EPSG:4326",
+            pixel: event.containerPoint
```

Comment:
That's the smaller part, the bigger part is figuring out how to write a useful test about this ;)